tactics/class_tactics.ml

```diff
@@ -258,7 +258,7 @@ and e_my_find_search db_list local_db secvars hdc complete only_classes env sigm
   let allowed_evars =
     let all = Evarsolve.AllowedEvars.all in
     match hdc with
-    | Some (hd,_) when only_classes ->
```

Comment:
I think this is the only use of `only_classes` so the argument should be removed.

Reply:
No, it's used further down. I don't think dune would even let me compile a function with an unused argument..

Reply:
Indeed, it's not really used. I have a fixed version but I'll let CI finish before I push.

Reply:
CI finished successfully. I pushed the removal of the argument.